Backend/api/controllers/PatientController.js

```diff
@@ -1,39 +1,45 @@
 // controllers/patientController.js
-import { Patient } from '../models/Patient.js';
-import bcrypt from 'bcrypt';
-import jwt from 'jsonwebtoken';
+import { Patient } from "../models/Patient.js";
+import bcrypt from "bcrypt";
+import jwt from "jsonwebtoken";
 
 // Register a new patient
 export const registerPatient = async (req, res) => {
   try {
-    const { name, email, gender, mobile, password } = req.body;
+    const { name, email, gender, mobile, password, age} = req.body;
```

Comment:
Remove the password field when adding a new patient. Replace the password field with an ID Number field instead.

Reply:
So how do you want to create the Id for the parent?

Reply:
Please tell me this as I also want to work on some other issue in your project.

Reply:
@sujal1256 This is the admin panel and we will be creating a new page specifically for adding patients. In the admin panel, patients will be added using their ID number not by their password.

Reply:
Ok so I am creating a unique Id for the patient using a npm library, and there will no password comming from frontend.